hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/terasort/TeraStreamValidate.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.pravega.example.hadoop.terasort;
+
+import io.pravega.connectors.hadoop.EventKey;
+import io.pravega.connectors.hadoop.PravegaInputFormat;
+import io.pravega.example.hadoop.wordcount.TextSerializer;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.conf.Configured;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapreduce.*;
+import org.apache.hadoop.mapreduce.lib.output.FileOutputFormat;
+import org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat;
+import org.apache.hadoop.util.Tool;
+import org.apache.hadoop.util.ToolRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ *  The class to validate terasort results.
+ *  It simply dumps the events of specified Pravega stream into hdfs file specified.
+ */
+public class TeraStreamValidate extends Configured implements Tool {
+  private static final Logger LOG = LoggerFactory.getLogger(TeraStreamValidate.class);
+
+  private static void usage() throws IOException {
+    System.err.println("Usage: terastreamvalidate [-Dproperty=value] " +
```

Comment:
Can you elaborate on what possible values can be passed for this `[-Dproperty=value]`?

Reply:
These are just regular Hadoop job properties, e.g. number of mappers, number of reducers etc

Reply:
does it make sense to document some of the properties that may be useful?

Reply:
Yes it helps to list a few properties for example, but I think it's ok too without it.